src/ProjectService.Mappers/ResponsesMappers/Interfaces/IRolesResponseMapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Responses;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.ResponsesMappers.Interfaces
+{
+    public interface IRolesResponseMapper
+    {
+        RolesResponse Map(IEnumerable<DbRole> dbRoles);
```

Comment:
need interface IMapper

Reply:
Added

Reply:
Your `IRolesResponseMapper` interface now inherits from the `IMapper` interface, which contains a `RolesResponse Map(IEnumerable<DbRole> dbRoles);` member. So it is already in the `IRolesResponseMapper`. 

So remove `RolesResponse Map(IEnumerable<DbRole> dbRoles);`